hassio/addon-view/hassio-addon-info.html

```diff
@@ -235,6 +238,11 @@
     const data = { auto_update: !this.addon.auto_update };
     this.hass.callApi('POST', `hassio/addons/${this.addonSlug}/options`, data);
   }
+
+  openChangelog() {
+    this.hass.callApi('get', `hassio/addons/${this.addonSlug}/changelog`)
+      .then(data => alert(data));
```

Comment:
instead a js alert, we can show a UI like snapshot and show the markdown inside this?